test/onnx/test_fx_to_onnx.py

```diff
@@ -209,15 +209,29 @@ def forward(self, input):
             expected_node="aten.convolution.default",
         )
 
-    # TODO: When registry is public, add a custom op cases to replace
-    # aten::add
     def test_dispatch_overload_fall_back_default_raise_diagnostic_warning(self):
         class TraceModel(torch.nn.Module):
             def forward(self, input):
-                return torch.ops.aten.add(input, input)
+                return torch.ops.aten.add.Tensor(input, input)
+
+        # _DEFAULT_OPSET_VERSION = 18
+        onnx_registry = torch.onnx.OnnxRegistry()
+        self.assertTrue(
```

Comment:
@BowenBao @justinchuby Why is this False? It works at local...

Reply:
I would avoid reliance on init behavior here. Potentially register an op to it, then check existence. In the future I suggest we don't populate the registry with `OnnxRegistry()` init alone.

Reply:
onnxscript version diff I suppose?

Reply:
> onnxscript version diff I suppose?

I see. That's it.

Reply:
> I would avoid reliance on init behavior here. Potentially register an op to it, then check existence. In the future I suggest we don't populate the registry with `OnnxRegistry()` init alone.

Could you share more of the thoughts? Creating OnnRegistry() which is initiated with full torchlib is kind of the core idea of making registry public. What is the other ways around?

Reply:
@justinchuby do you mean lazy population of torchlib registry?

🤔 I think it can work, bottom line is we hide the implementation detail and only lazy populate when needed (read) so the behavior stays the same. 

But I'm not sure what are we gaining? We can make torchlib registry read only too.

Reply:
I just think for this test to be robust it would be nice to not assume the existence of an op from torchlib.

Reply:
okay :)

Reply:
Agree, I was going to use pure custom ops from torch, but torch only supports custom op with namespace and op_name, so I can't have it with overloads. I already put TODO on that.